release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -3315,3 +3315,51 @@ You can view the container images in this release by running the following comma
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.16.1 --pullspecs
+----
+
+[id="ocp-4-16-1-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, an error in `growpart` caused the device to be locked, which prevented the Linux Unified Key Setup-on-disk-format (LUKS) device being opened. The resulted was that the system was unable to boot and went into emergency mode. With this release, the call to the `growpart` is removed and this issue is fixed. (link:https://issues.redhat.com/browse/OCPBUGS-35973[*OCPBUGS-35973*])
+
+* Previously, a bug in systemd might cause the `coreos-multipath-trigger.service` unit to hang forever. As a result the system would never finish booting. With this release, the systemd unit was removed and the issue is fixed. (link:https://issues.redhat.com/browse/OCPBUGS-35748[*OCPBUGS-35748*])
+
+* Previously, the KMS key was applied as an empty string. This always applied a KMS key that was invalid. With this release, the empty string is removed and the KMS key is only applied when one exists from the `install-config.yaml`. (link:https://issues.redhat.com/browse/OCPBUGS-35531[*OCPBUGS-35531*])
+
+* Previously, there was no validation of the values for confidential compute and on host maintenance set by the user. With this release, when confidential compute is enabled by the user the value for `onHostMaintenance` must be set to `onHostMaintenance: Terminate`. (link:https://issues.redhat.com/browse/OCPBUGS-35493[*OCPBUGS-35493*])
+
+* Previously, in User Provisioned Infrastructure (UPI) or clusters that were upgraded from older versions, `failureDomains` may be missing in Infrastructure objects which caused certain checks to fail. With this release, a fallback `failureDomains` is synthesized from `cloudConfig` if none are available in `infrastructures.config.openshift.io`. (link:https://issues.redhat.com/browse/OCPBUGS-35446[*OCPBUGS-35446*])
+
+* Previously, when a new version of a Custom Resource Definition (CRD) specified a new conversion strategy, this conversion strategy was expected to successfully convert resources. This was not the case because Operator Lifecycle Manager (OLM) cannot run the new conversion strategies for CRD validation without actually performing the update operation. With this release, the OLM implementation generates a warning message during the update process when CRD validations fail with the existing conversion strategy and the new conversion strategy is specified in the new version of the CRD. (link:https://issues.redhat.com/browse/OCPBUGS-35373[*OCPBUGS-35373*])
```

Comment:
```suggestion
* Previously, when a new version of a custom resource definition (CRD) specified a new conversion strategy, this conversion strategy was expected to successfully convert resources. This was not the case because Operator Lifecycle Manager (OLM) cannot run the new conversion strategies for CRD validation without actually performing the update operation. With this release, the OLM generates a warning message during the update process when CRD validations fail with the existing conversion strategy and the new conversion strategy is specified in the new version of the CRD. (link:https://issues.redhat.com/browse/OCPBUGS-35373[*OCPBUGS-35373*])
```